src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,724 @@
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import { ErrorView } from '@app/ErrorView/ErrorView';
+import LoadingView from '@app/LoadingView/LoadingView';
+import {
+  automatedAnalysisAddFilterIntent,
+  automatedAnalysisAddTargetIntent,
+  automatedAnalysisDeleteAllFiltersIntent,
+  automatedAnalysisDeleteCategoryFiltersIntent,
+  automatedAnalysisDeleteFilterIntent,
+} from '@app/Shared/Redux/AutomatedAnalysisFilterActions';
+import { TargetAutomatedAnalysisFilters } from '@app/Shared/Redux/AutomatedAnalysisFilterReducer';
+import { RootState, StateDispatch } from '@app/Shared/Redux/ReduxStore';
+import { ArchivedRecording, automatedAnalysisRecordingName, Recording } from '@app/Shared/Services/Api.service';
+import {
+  FAILED_REPORT_MESSAGE,
+  INTERNAL_ERROR_MESSAGE,
+  NO_RECORDINGS_MESSAGE,
+  RECORDING_FAILURE_MESSAGE,
+  RuleEvaluation,
+} from '@app/Shared/Services/Report.service';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { automatedAnalysisConfigToRecordingAttributes } from '@app/Shared/Services/Settings.service';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardExpandableContent,
+  CardHeader,
+  CardTitle,
+  Checkbox,
+  Grid,
+  GridItem,
+  LabelGroup,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  Tooltip,
+} from '@patternfly/react-core';
+import { PlusCircleIcon, Spinner2Icon, TrashIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { first, map, tap } from 'rxjs';
+import { AutomatedAnalysisConfigDrawer } from './AutomatedAnalysisConfigDrawer';
+import {
+  AutomatedAnalysisFilters,
+  AutomatedAnalysisFiltersCategories,
+  AutomatedAnalysisGlobalFiltersCategories,
+  emptyAutomatedAnalysisFilters,
+  filterAutomatedAnalysis,
+} from './AutomatedAnalysisFilters';
+import { clickableAutomatedAnalysisKey, ClickableAutomatedAnalysisLabel } from './ClickableAutomatedAnalysisLabel';
+import { AutomatedAnalysisScoreFilter } from './Filters/AutomatedAnalysisScoreFilter';
+
+interface AutomatedAnalysisCardProps {
+  pageTitle: string;
```

Comment:
The title is pretty much unique for this card. Maybe we can just directly use it instead of props?

Reply:
I don't think I was using this anyways, haha I'll just remove it.